README.md

```diff
@@ -31,16 +31,18 @@ Prerequisites
 ---------
  * [Java 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (Java 9+ are not supported, will get compiler issue)
  * [Git](https://git-scm.com/downloads)
+ * [Gradle](https://gradle.org/install/)
 
 Building from source code
 ---------
- Clone the repository using Git.
+ Clone the repository using Git and go to the root directory
  ```
- git clone git@github.com:vmware/singleton.git
+ git clone https://github.com/vmware/singleton.git
```

Comment:
Maybe it is better to show the way to clone by both https and ssh.

Reply:
ok that's much better